build-system/npm-publish/write-package-files.js

```diff
@@ -153,27 +154,34 @@ async function writePackageJson() {
     exports[`./${stylesheet}`] = `./dist/${stylesheet}`;
   }
 
+  const staticPackage = JSON.parse(
+    await readFile(`${dir}/package.static.json`, 'utf8')
+  );
+
   const json = {
     name: `@bentoproject/${packageName}`,
-    version,
-    description: await getDescription(),
     author: 'Bento Authors',
+    files: ['dist/*', 'react.js', 'styles.css'],
+    ...staticPackage,
     license: 'Apache-2.0',
     main: './dist/web-component.js',
     module: './dist/web-component.module.js',
-    exports,
-    files: ['dist/*', 'react.js', 'styles.css'],
     repository: {
       type: 'git',
       url: 'https://github.com/ampproject/amphtml.git',
       directory: dir,
     },
     homepage: `https://github.com/ampproject/amphtml/tree/main/${dir}`,
+    version,
+    exports,
+    description: await getDescription(),
     peerDependencies: {
+      ...(staticPackage.peerDependencies || {}),
       preact: '^10.2.1',
       react: '^17.0.0',
     },
     dependencies: {
+      ...(staticPackage.dependencies || {}),
```

Comment:
I know pnpm supports a special syntax for monorepos called `workspace` that forces pnpm to resolve only packages in the workspace: https://pnpm.io/workspaces#workspace-protocol-workspace. Setting the version of a dependency to `workspace:*` makes pnpm resolve the current version of a local package. Maybe we can adopt a similar syntax for this purpose?

Reply:
I've added support for a constant `AMP_VERSION` though I'm not very happy about the name and am happy for suggestions. Note that we shouldn't need to support the syntax for non exact versions.

Reply:
small nit: BENTO_VERSION might be preferable to AMP_VERSION since all of this is for the npm package within `@bentoproject/`, but sounds good! 

As far as other suggestions, it yarn v2 also supports the workspace:* syntax (https://yarnpkg.com/features/workspaces#workspace-ranges-workspace), so that might be a good convention to follow. I'm fine with how it is though.

Reply:
`BENTO_VERSION` definitely makes sense. `workspace:*` seems to imply matching multiple versions which we do not support. If that changes I'm all on board for making the change though.